tests/shell/pipeline/commands/external.rs

```diff
@@ -217,6 +218,27 @@ mod external_words {
 
         assert_eq!(actual.out, "test \"things\"");
     }
+
+    #[test]
+    fn external_arg_with_quotes() {
+        Playground::setup("external_arg_with_quotes", |dirs, sandbox| {
+            sandbox.with_files(vec![FileWithContent(
+                "sample.toml",
```

Comment:
@jonathandturner, I'm wondering what happens if the path contains spaces? Or do you tackle this case in another issue?

Reply:
Paths with spaces should work. I've done a few completion updates, which includes improving how we handle paths with spaces.

Are you seeing any issues with the latest PRs?

Reply:
My thought was that the test could reflect that spaces are supported. Like that:

```suggestion
                "a sample file.toml",
```

So, I don't see any trouble just an slight test code improvement.

Reply:
@schrieveslaach if you're interested, we'd love to have more tests to help cover cases like that.

Reply:
I'll take a look. 👍

Reply:
@jonathandturner, I created #3596, do you mind and have a look at it?